src/energy_comms/extract/qcew_2014_2021.py

```diff
@@ -0,0 +1,180 @@
+# imports
```

Comment:
I think all these QCEW extracts can live in the same module because they're all the same source. And then have different functions and parameters the different years of data.